synapse/lib/view.py

```diff
@@ -113,6 +122,56 @@ async def __anit__(self, core, node):
         # isolate some initialization to easily override.
         await self._initViewLayers()
 
+        self.trigtask = None
+        await self.initTrigTask()
+
+    async def initTrigTask(self):
+
+        if self.trigtask is not None:
+            return
+
+        if not await self.core.isCellActive():
+            return
+
+        self.trigtask = self.schedCoro(self._trigQueueLoop())
+
+    async def finiTrigTask(self):
+
+        if self.trigtask is not None:
+            self.trigtask.cancel()
+            self.trigtask = None
+
+    async def _trigQueueLoop(self):
+
+        while not self.isfini:
+
+            async for offs, triginfo in self.trigqueue.gets(0):
```

Comment:
If we're not using wait=true we should add a self.waitfini(1) at the end of while loop?

Reply:
Whups! Should have been wait=true.  Will fix, thanks!

Reply:
Wait...  SlabSeqn.gets() has wait=true by default

Reply:
I thought this was a MultiQueue object :laughing: